src/test/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGeneratorTest.java

```diff
@@ -39,42 +40,46 @@ public void before() {
         currTime = System.currentTimeMillis();
     }
 
-    @Test
-    public void getSonarProjectNameFromNewSonarQube() throws Exception {
-        String name = "org.namespace:feature%2Fmy-sub-project";
-        String url = sonarUrl + "/dashboard?id=" + name;
-        SonarQubePointGenerator gen = new SonarQubePointGenerator(build, listener, measurementRenderer, currTime, StringUtils.EMPTY, CUSTOM_PREFIX);
-        assertEquals(gen.getSonarProjectName(url), name);
-    }
+    // @Test
+    // public void getSonarProjectNameFromNewSonarQube() throws Exception {
+    //     EnvVars envVars = new EnvVars();
+    //     String name = "org.namespace:feature%2Fmy-sub-project";
+    //     String url = sonarUrl + "/dashboard?id=" + name;
+    //     SonarQubePointGenerator gen = new SonarQubePointGenerator(build, listener, measurementRenderer, currTime, StringUtils.EMPTY, CUSTOM_PREFIX, envVars);
+    //     assertEquals(gen.getSonarProjectName(url), name);
+    // }
```

Comment:
Removed. Initially I commented out this test because the function being tested, getSonarProjectName, is not necessary any more. The project name is easily retrieved from the sonar build report.